### DISCUSSION TITLE
[Feature]: Bilder im Banner: Verlinkung ermöglichen

### DISCUSSION THREAD

**Discussion by wie0815** [lotta-schule/web#266] [Ideen]
### Bereich

Administrationsbereich (inkl. "Feedback")

### Verhalten

Banner sind statische Überschriften mit einem Hintergrundbild

### Problem

Unsere Schule hat in jedem Jahr 2-3 Veranstaltungen, die sehr öffentlichkeitswirksam sind. Lehrer haben die Idee geäußert, dass temporär ein Banner auf der Startseite auf solche Ereignisse aufmerksam machen kann. (mit Lotta kein Problem)

Da täglich viele Beiträge auf der Startseite "nach oben rutschen" kommt es sehr oft vor, dass Beiträge übersehen werden. So zum Beispiel der Beitrag zum Erwerb einer Eintrittskarte für die Oskarverleihung.

### Beschreibung

Wir führen temporäre Banner ein: ein Klick auf den Banner führt den Besucher zu einem verlinkten Beitrag.

Grundsätzlich können Administratoren entscheiden, ob vom Banner ein Beitrag verlinkt wird.

**Comment by techotline** [comment]
Mir gefällt die Idee. Interaktive Banner wären ein Mehrwert, um den Workflow mit der jeweiligen Seite zu stärken. Insbesondere an einer großen Schule werden über den Tag zahlreiche Beiträge aktualisiert. Weiterhin sollte das Prinzip der Konfiguration unbedingt erhalten bleiben und das Tool für Verlinkung im Admin-Bereich unter der Einstellungen für den Banner platziert werden. Ggf. nutzen wir perspektivisch die Checkboxen für die Erweiterungen. Die Frage wird nur sein, ob das Priorität hat und welcher Aufwand betrieben werden muss, um es umzusetzen. Sollten wir zeitnah aufgreifen und klären. 😊

**Comment by ptitmouton** [comment]
Vermutlich lösbar mit sinnvoller Umsetzung von #267
